src/Markup/Perspex.Markup/Data/Plugins/ValidationCheckerBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Perspex.Data;
+
+namespace Perspex.Markup.Data.Plugins
+{
+    public abstract class ValidationCheckerBase : IPropertyAccessor
```

Comment:
The name `ValidationCheckerBase` suggests that it will be a base class for e.g. `ExceptionValidationCheckerPlugin`, but it's not... I think a better name is needed here.